com.unity.render-pipelines.universal/Runtime/NativeRenderPass.cs

```diff
@@ -339,41 +307,23 @@ internal void SetNativeRenderPassAttachmentList(ScriptableRenderPass renderPass,
 
                     AttachmentDescriptor currentAttachmentDescriptor;
                     var usesTargetTexture = cameraData.targetTexture != null;
-                    var depthOnly = renderPass.depthOnly || (usesTargetTexture && IsDepthOnlyRenderTexture(cameraData.targetTexture));
-                    // Offscreen depth-only cameras need this set explicitly
-                    if (depthOnly && usesTargetTexture)
+                    var depthOnly = (pass.colorAttachmentHandle.rt != null && IsDepthOnlyRenderTexture(pass.colorAttachmentHandle.rt)) || (usesTargetTexture && IsDepthOnlyRenderTexture(cameraData.targetTexture));
+
+                    int samples;
+                    RenderTargetIdentifier colorAttachmentTarget;
+                    // We are not rendering to Backbuffer so we have the RT and the information with it
+                    if (passColorAttachment.nameID != BuiltinRenderTextureType.CameraTarget)
```

Comment:
You might have an issue with XR slice being -1 and not being caught here.

Reply:
will change this to passColorAttachment != kCameraTarget. Should that take care of the check?

Reply:
I don't think it will. You're better off using `new RenderTargetIdentifier(passColorAttachment.nameId, 0, slice=0)`

Reply:
Done in eb5919e